lib/node/runner/checker.go

```diff
@@ -376,6 +464,11 @@ func FinishedBallotStore(c common.Checker, args ...interface{}) (err error) {
 		return
 	}
 	if checker.FinishedVotingHole == ballot.VotingYES {
+		if err = getMissingTransaction(checker); err != nil {
```

Comment:
Is there a chance that any Tx are missing when the ballot passed the consensus?
@spikeekips

Reply:
@kfangw It will be rarely occurred, but yes.

1. One node failed to receive the ballots of previous state.
1. The node received the ballots of ACCEPT state.
1. The node must confirm the ACCEPTed ballot, but the node does not know about the transactions of ACCEPTed ballot.

Reply:
Understood. Thx